coorx/base_transform.py

```diff
@@ -72,8 +79,9 @@ def __init__(self, dims:Dims=None, from_cs:StrOrNone=None, to_cs:StrOrNone=None,
         self._change_callbacks = []
         self._systems = (None, None)
 
-        # optional coordinate system tracking
-        self.set_systems(from_cs, to_cs, cs_graph)
+        with contextlib.suppress(NotImplementedError):
+            # optional coordinate system tracking
+            self.set_systems(from_cs, to_cs, cs_graph)
```

Comment:
We could probably make this an `if not self.Dependent`, to be clearer.

Reply:
Not a fan of silently ignoring arguments. Also, I can hink of cases where one transform is dependent on another, but doesn't know what systems it maps between.

Reply:
Exception: raised.

The only two dependent classes we have are Inverse and Composite, and those always have dependent coordinate systems, do they not? The only exceptional case I can think of is if a transform is not totally invertible, in which case... no, I would just want to error when creating the inverse in the first place, not make a partial inverse or some such. Can you describe a situation where a dependent transform has coordinate systems distinct from its dependencies? Is it a third class of dependent transform?